pkg/sql/ir_generator.go

```diff
@@ -59,12 +59,13 @@ func generateTrainIR(slct *extendedSelect, connStr string) (*codegen.TrainIR, er
 			Name: tc.label,
 		}}
 
-	// TODO(typhoonzero): fill in ValidationSelect using `create_train_val.go`
-	// TODO(typhoonzero): fill in ValidationSelect when VALIDATE clause is ready
+	validationDS, _ := parseValidataionDataset(attrList)
```

Comment:
Naming: Does `DS` refer to `Dataset`? If so, maybe change to `validationDataset` or `validationSelect`.